cms/templates/home_page.html

```diff
@@ -31,7 +31,7 @@ <h1>{{ product_cards_section_title }}</h1>
     <ul class="course-listing card-listing">
       {% for product in products %}
       <li>
-        <div class="course-card highlight-card">
+        <div onclick="location='{{ product.url_path }}'" class="course-card highlight-card">
```

Comment:
For accessibility reasons and a few others, we should implement this with a normal `<a>` tag. You can either replace this `<div>` with an `<a>`, or add the `<a>` as the first child of this div